common/static/js/capa/src/formula_equation_preview.js

```diff
@@ -3,6 +3,37 @@ var formulaEquationPreview = {
     errorDelay: 1500  // Wait time before showing error (prevent frustration).
 };
 
+function escapeCSS(id) {
+    // Wrapper around window.CSS.escape that uses a fallback method if CSS.escape is not available.
+
+    // TODO: if this escaping is also required elsewhere, it may be useful to add a global CSS.escape polyfill and
+    // use that directly.
+    if (window.CSS && window.CSS.escape) {
+        return CSS.escape(id);
+    } else {
+        // CSS escape alternative borrowed from jQuery.escapeSelector (new in version 3; this whole block can be
+        // replaced once we upgrade).
```

Comment:
Maybe add a link to the code this was borrowed from?

Reply:
Done. :)